libraries/libfc/src/log/logger.cpp

```diff
@@ -82,14 +81,14 @@ namespace fc {
        }
     }
 
-    void logger::set_name( const fc::string& n ) { my->_name = n; }
-    const fc::string& logger::name()const { return my->_name; }
+    void logger::set_name( const std::string& n ) { my->_name = n; }
```

Comment:
No need to change, but I think a good convention for methods accepting a potentially moveable object that is to be immediately copied, is to accept a `value` parameter and `std::move` it where it should go.

Reply:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rf-in

Reply:
Logger hopefully will be replaced soon by a third-party logging library.

Reply:
Sounds good to me!